nextcord/reaction.py

```diff
@@ -92,6 +139,22 @@ def __str__(self) -> str:
     def __repr__(self) -> str:
         return f"<Reaction emoji={self.emoji!r} me={self.me} count={self.count}>"
 
+    @cached_property
+    def burst_colours(self) -> List[Colour]:
+        """List[:class:`Colour`]: The HEX colors used for a burst reaction.
```

Comment:
I'm confused on whether we're sticking with American or British English here. Also why is hex in uppercase?

Reply:
Traditionally, there is a function with the British English name that holds the functionality and an American alias for it. An example would be [User.colour and User.color](https://github.com/nextcord/nextcord/blob/master/nextcord/user.py#L215-L231).

Reply:
I'm aware of this, but the problem is American English spelling is used in this docstring. In the example you brought up, the proper spellings for the dialects are used.

Reply:
My mistake British English it is. Also "HEX" is capitalised for no reason in particular really, AFAIK you can write it capitalised or uncapitalised